### ISSUE TITLE
Wrong value copied

### ISSUE THREAD

**Issue by andreax79** [sasaplus1/deepcopy.js#9]
With deepcopy 0.6.0 and Node v0.12.7 on OS/X,  the following code:

```
var deepcopy = require('deepcopy');

var b = [
  {
    one: true,
    two: 'endpoint C',
  },
  {
    one: true,
    two: 'endpoint D',
  }
];

console.log('original');
console.log(b);
console.log('copied');
console.log(deepcopy(b));
```

produces the following output:

```
original
[ { one: true, two: 'endpoint C' }, { one: true, two: 'endpoint D' } ]
copied
[ { one: true, two: 'endpoint C' }, { one: true, two: true } ]
```

**Comment by sasaplus1** [comment]
@andreax79

Thanks for your bug report. I will fix it later.
If you are hurry, please use deepcopy@0.5.0.

**Comment by sasaplus1** [comment]
fixed version 0.6.1.
